### PR TITLE
#1595: Linked team names to detail pages in admin tools

### LINKED ISSUE #1595: [Frontend] - Teams on the Bottom of the Admin Tool Page should link to the Teams Page
### Description

We have a list of teams on the bottom of the admin tools page next to the create team Admin Tool. We wanna make all the teams in this list clickable and link to the team detail page for that team when you click them.

### Acceptance Criteria

- [x] When you click on a team in the list of teams on the admin tools page, it takes you to the detail page for that team

### Proposed Solution

See how we do it for the CR/ Projects tables

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by arinjay-singh**
## Changes

Linked team names to respective pages, so when you click on a team in the list of teams on the admin tools page it takes you to the detail page for that team.

## Notes

No notes.

## Test Cases

- Tested for each of the mock teams.

## To Do

No to-dos.

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1595

### CODE REVIEW

**Review by Aaryan1203 — DISMISSED** [review]
Great work! Next time try to add a screen recording for front-end changes if you can!

**Comment by arinjay-singh** [comment]
I didn't think to add screenshots to the pr because there were no visible changes.

File: src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx [hunk 1]
```diff
@@ -79,7 +81,11 @@ const TeamsTools = () => {
 
   const teamTableRows = allTeams.map((team) => (
     <TableRow>
-      <TableCell sx={{ border: '2px solid black' }}>{team.teamName}</TableCell>
+      <TableCell sx={{ border: '2px solid black' }}>
```
Comment (RChandler234) [inline src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx:84]: clicking anywhere on the row should link to the team page, not just this cell

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx (hunk 1 above)
Comment (arinjay-singh) [reply src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx]: Just pushed another commit with the fix. Sorry about that.

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**